pkg/app/logging_windows.go

```diff
@@ -0,0 +1,95 @@
+// +build windows
+
+package app
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/aybabtme/humanlog"
+	"github.com/jesseduffield/lazygit/pkg/config"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+func TailLogs() {
+	logFilePath, err := config.LogPath()
```

Comment:
I feel like `TailLogs` is fine to stay where it was, but `TailLogsNative` should be moved into a separate file, given that it's only in that function where we see differences in behaviour between different OS's

Reply:
ok, so this is probably me missing something to do with conditional compilation: as far as I understand, conditional compilation is working here on a per-file basis, so I thought I'd conditionally compile `TailLogs`, unless you'd like me to rather only conditionally compile `TailLogsNative` and take the body of the original `TailLogs`, after the original guard checks, and create a !windows `TailLogsNative` which does that? It's certainly less duplication of code. I'll tentatively push that up, and see what you think - I can always revert.